# HG changeset patch
# User Julien Cristau <jcristau@debian.org>
# Date 1593076954 -7200
#      Thu Jun 25 11:22:34 2020 +0200
# Node ID 39b6a9ff4c91b1b3a0e9d2eade46b6bf3d8f7d3c
# Parent  a46e36b824612be16b38a1c1cd11160c293d115b
test: redirect stderr so warning messages don't cause changed output (issue6237)

clone and commit race for the lock, and if commit has to wait more than
a second it prints a warning to stderr.  Since this is somewhat expected
here, silence it.

Differential Revision: https://phab.mercurial-scm.org/D8664

diff --git a/tests/test-clone-uncompressed.t b/tests/test-clone-uncompressed.t
--- a/tests/test-clone-uncompressed.t
+++ b/tests/test-clone-uncompressed.t
@@ -407,7 +407,7 @@ actually serving file content
   $ sleep 1
   $ echo >> repo/f1
   $ echo >> repo/f2
-  $ hg -R repo ci -m "1"
+  $ hg -R repo ci -m "1" 2>/dev/null
   $ wait
   $ hg -R clone id
   000000000000
